src/pages/settings/PreferencesPage.js

```diff
@@ -38,42 +48,71 @@ const priorityModes = {
     },
 };
 
-const PreferencesPage = ({priorityMode}) => (
-    <>
-        <HeaderGap />
-        <HeaderWithCloseButton
-            title="Preferences"
-            shouldShowBackButton
-            onBackButtonPress={() => redirect(ROUTES.SETTINGS)}
-            onCloseButtonPress={() => redirect(ROUTES.HOME)}
-        />
-        <View style={styles.pageWrapper}>
-            <View style={[styles.settingsPageBody, styles.mb6]}>
-                <Text style={[styles.formLabel]} numberOfLines={1}>
-                    Priority Mode
-                </Text>
-                <View style={[styles.mb2]}>
-                    {/* empty object in placeholder below to prevent default */}
-                    {/* placeholder from appearing as a selection option. */}
-                    <RNPickerSelect
-                        onValueChange={
-                            mode => NameValuePair.set(CONST.NVP.PRIORITY_MODE, mode, ONYXKEYS.PRIORITY_MODE)
-                        }
-                        items={Object.values(priorityModes)}
-                        style={styles.picker}
-                        useNativeAndroidPickerStyle={false}
-                        placeholder={{}}
-                        value={priorityMode}
-                        Icon={() => <Icon src={DownArrow} />}
-                    />
+class PreferencesPage extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            isNewsStatusEnabled: !(this.props.user.expensifyNewsStatus === false),
+        };
+        this.toggleNewsStatus = this.toggleNewsStatus.bind(this);
+    }
+
+    toggleNewsStatus(toggle) {
+        this.setState({isNewsStatusEnabled: toggle});
+        setExpensifyNewsStatus(toggle);
+    }
+
+    render() {
+        return (
+            <>
+                <HeaderGap />
+                <HeaderWithCloseButton
+                    title="Preferences"
+                    shouldShowBackButton
+                    onBackButtonPress={() => redirect(ROUTES.SETTINGS)}
+                    onCloseButtonPress={() => redirect(ROUTES.HOME)}
+                />
+                <View style={styles.pageWrapper}>
+                    <View style={[styles.settingsPageBody, styles.mb6]}>
+                        <View style={[styles.flexRow, styles.mb5, styles.justifyContentBetween]}>
```

Comment:
Since we now have multiple items in the Preferences section, I think it might be nice if each item has a little header. 

So could we see it where we add `<Text style={[styles.formLabel]} numberOfLines={1}>Notifications</Text>` above this particular toggle?